iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/pipe/connector/protocol/IoTDBConfigRegionConnector.java

```diff
@@ -63,14 +63,16 @@ protected IoTDBSyncClientManager constructClient(
       final String trustStorePwd,
       final boolean useLeaderCache,
       final String loadBalanceStrategy,
-      final boolean shouldReceiverConvertOnTypeMismatch) {
+      final boolean shouldReceiverConvertOnTypeMismatch,
+      final String loadTsFileStrategy) {
     return new IoTDBConfigNodeSyncClientManager(
         nodeUrls,
         useSSL,
         trustStorePath,
         trustStorePwd,
         loadBalanceStrategy,
-        shouldReceiverConvertOnTypeMismatch);
+        shouldReceiverConvertOnTypeMismatch,
+        loadTsFileStrategy);
```

Comment:
Can pass "sync" here, users may use "async-load" in "inclusion=all", then the configConnector will pass the "async-load" to receiver.... And it'll cause problem when handling the snapshot.